.github/workflows/test-the-tests.yml

```diff
@@ -26,3 +26,9 @@ jobs:
         TESTING_ENV: 'dev'
       with:
         args: yarn run test-ci safe-web/read_only
+    - name: Upload artifact
+      if: ${{ always() }}
```

Comment:
Are we sure we want to record and store the .mp4 at every run? What do you think of saving it conditionally? https://github.com/actions/upload-artifact#conditional-artifact-upload

Reply:
do you mean upload the file only if the test fails? Idk if we can know that (if there is a flag or something we can check if the test has fail at some point). Also as far as I know, the files will be over written and we can make something that clean those files after a few days or so.
cc @DaniSomoza

Reply:
yep, from my side I think that makes sense to upload it only on failure executions:

```
    if: failure()
```